core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage.properties

```diff
@@ -0,0 +1,33 @@
+title=Manage Legacy API Token usage
+contextMessage=The following users have a legacy API token. \
+  Because legacy tokens are stored in a recoverable format, we recommend migrating to the new API token system. \
+  Additionally, in previous version of Jenkins, API tokens were automatically created for every new user. \
+  Often, users did not use the token, resulting in a larger attack surface than necessary. \
+  Automatic token generation must now be enabled explicitly.
+recommendationMessage=For such reasons, we recommend to: \
+<ul>\
+  <li>revoke the tokens that were never used and</li>\
+  <li>ask users that are currently using their legacy token to generate a token using the new system and revoke their legacy token</li>\
+</ul>
+UserId=User Id
+UserFullName=User full name
+TokenName=Token name
+NumDaysSinceCreation=Days since creation
+NumOfUse=# of use
+NumDaysSinceLastUse=Days since last use
+HasFreshToken=Fresh token?
+HasFreshToken_tooltip=A fresh token is one that was created using the new system\n\
+  after the most recent use of the user's legacy token. \n\n\
+  Note: the creation date of legacy tokens which have never been used are reset every time Jenkins is restarted, \n\
+  which means that the date may be inaccurate.
+HasMoreRecentlyUsedToken=Recent token?
+HasMoreRecentlyUsedToken_tooltip=A recently used token is one that was created using the new system \n\
+  and used more recently than the user's legacy token. \n\n\
+  Note: the creation date of legacy tokens which have never been used are reset every time Jenkins is restarted, \n\
+  which means that the date may be inaccurate.
```

Comment:
Maybe it's just me, but these extra line breaks look weird on my screen. They disrupt the reading flow. Generally it's better to let the browser figure out where to wrap lines. Except where the line breaks are productive to the flow of the information. In other words, the line breaks before "Note:" are good and improve clarity.

Comment applies to both of these tooltips.